dusty/cli/disk.py

```diff
@@ -24,3 +30,13 @@ def main(argv):
         return Payload(cleanup_inactive_containers)
     elif args['cleanup_images']:
         return Payload(cleanup_images)
+    elif args['backup']:
+        path = os.path.abspath(args['<destination>'])
+        return Payload(backup, path)
+    elif args['restore']:
+        path = os.path.abspath(args['<source>'])
+        print "Warning: this will overwrite the /persist directory on your VM with the contents of {}".format(path)
+        if raw_input("Continue? (y/n) ").strip().upper() == 'Y':
+            return Payload(restore, path)
+        else:
+            print "Restore cancelled"
```

Comment:
just the standard way we send messages to users throught the daemon and the client